tasks_management/signals.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+
+from core.service_signals import ServiceSignalBindType
+from core.signals import bind_service_signal
+from tasks_management.models import TaskGroup, Task
+
+
+def resolve_task_any(task, business_status: Dict[str, str]):
+    if any(status == "APPROVED" for status in business_status.values()):
+        return True
+    return False
+
+
+def resolve_task_all(task, business_status: Dict[str, str]):
+    approve_count = sum(1 for status in business_status.values() if status == "APPROVED")
+    group_members = TaskGroup.objects.get(task=task).taskexecutor_set.count()
+    if approve_count == group_members:
+        return True
+    return False
+
+
+def resolve_task_n(task, business_status: Dict[str, str]):
+    approve_count = sum(1 for status in business_status.values() if status == "APPROVED")
+    n = 1  # hardcoded for now
+    if approve_count >= n:
+        return True
+    return False
+
+
+def executor_action_event_handler(**kwargs):
+    class_instance = kwargs.get("cls_")
+    data = kwargs.get("result").get("data")
+    task = Task.objects.get(id=data["task"]["id"])
```

Comment:
I think we don't have to requery the task as the ['data'] should already contain the `executor_action_event` and `business_status`, but that's not a blocking issue. Looks good to me

Reply:
Task is used by resolve functions, we could pass just an id but if there will be a case when more fields will be needed there won't be a need to extend arguments in those functions.